<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JPSeditor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Ressource.qrc">
    <normaloff>:/jupedsim.png</normaloff>:/jupedsim.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_ffnen_xml"/>
    <addaction name="action_ffnen"/>
    <addaction name="action_ffnen_cogmap"/>
    <addaction name="actionSpeichern"/>
    <addaction name="actionSpeichern_dxf"/>
    <addaction name="actionSpeichern_cogmap"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_ber"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSelect_Mode"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCopy"/>
    <addaction name="separator"/>
    <addaction name="actionWall"/>
    <addaction name="actionDoor"/>
    <addaction name="actionHLine"/>
    <addaction name="separator"/>
    <addaction name="actiongridmode"/>
    <addaction name="actionObjectsnap"/>
    <addaction name="actionanglesnap"/>
    <addaction name="actionDelete_lines"/>
    <addaction name="actionDelete_single_line"/>
    <addaction name="separator"/>
    <addaction name="actionRoom"/>
    <addaction name="actionLandmark"/>
    <addaction name="actionLandmarkWidget"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRotate_90_deg_clockwise"/>
    <addaction name="actionShow_Point_of_Origin"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect_Mode"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionWall"/>
   <addaction name="actionDoor"/>
   <addaction name="actionHLine"/>
   <addaction name="separator"/>
   <addaction name="actiongridmode"/>
   <addaction name="separator"/>
   <addaction name="actionObjectsnap"/>
   <addaction name="separator"/>
   <addaction name="actionanglesnap"/>
   <addaction name="separator"/>
   <addaction name="actionDelete_lines"/>
   <addaction name="separator"/>
   <addaction name="actionDelete_single_line"/>
   <addaction name="separator"/>
   <addaction name="actionRoom"/>
   <addaction name="separator"/>
   <addaction name="actionLandmarkWidget"/>
   <addaction name="actionLandmark"/>
  </widget>
  <action name="action_ffnen">
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>Load DXF</string>
   </property>
   <property name="toolTip">
    <string>Load DXF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/filesave.png</normaloff>:/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save as XML</string>
   </property>
   <property name="toolTip">
    <string>Save as XML</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/man-gimp1.png</normaloff>:/man-gimp1.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_ber">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actiongridmode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/Grid.png</normaloff>:/Grid.png</iconset>
   </property>
   <property name="text">
    <string>Gridmode</string>
   </property>
   <property name="toolTip">
    <string>Gridmode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionWall">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/Wall.png</normaloff>:/Wall.png</iconset>
   </property>
   <property name="text">
    <string>Wall</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionObjectsnap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/snap.png</normaloff>:/snap.png</iconset>
   </property>
   <property name="text">
    <string>Object snap</string>
   </property>
   <property name="toolTip">
    <string>Object snap</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="actionDelete_lines">
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/DeleteAll.png</normaloff>:/DeleteAll.png</iconset>
   </property>
   <property name="text">
    <string>Delete all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionDelete_single_line">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/Delete.png</normaloff>:/Delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete selected lines</string>
   </property>
   <property name="toolTip">
    <string>Delete selected lines</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionDoor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/Door.png</normaloff>:/Door.png</iconset>
   </property>
   <property name="text">
    <string>Door</string>
   </property>
   <property name="toolTip">
    <string>Door</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/Exit.png</normaloff>:/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/Room.png</normaloff>:/Room.png</iconset>
   </property>
   <property name="text">
    <string>Rooms</string>
   </property>
   <property name="toolTip">
    <string>Define Room</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSelect_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/basicCursor.png</normaloff>:/basicCursor.png</iconset>
   </property>
   <property name="text">
    <string>Select Mode</string>
   </property>
   <property name="toolTip">
    <string>Select Mode</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSpeichern_dxf">
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/filesave.png</normaloff>:/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save as DXF</string>
   </property>
   <property name="toolTip">
    <string>Save as DXF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="action_ffnen_xml">
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>Load XML</string>
   </property>
   <property name="toolTip">
    <string>Load XML</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+O</string>
   </property>
  </action>
  <action name="actionRotate_90_deg_clockwise">
   <property name="text">
    <string>Rotate (90 deg clockwise)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionLandmark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/statue.jpg</normaloff>:/statue.jpg</iconset>
   </property>
   <property name="text">
    <string>Landmark</string>
   </property>
   <property name="toolTip">
    <string>Landmark</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionLandmarkWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/statue_specs.jpg</normaloff>:/statue_specs.jpg</iconset>
   </property>
   <property name="text">
    <string>LandmarkSpecs</string>
   </property>
   <property name="toolTip">
    <string>LandmarkSpecs</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionanglesnap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/anglesnap.PNG</normaloff>:/anglesnap.PNG</iconset>
   </property>
   <property name="text">
    <string>Orthomode</string>
   </property>
   <property name="toolTip">
    <string>Orthomode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_ffnen_CogMap">
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>Load CognitiveMap</string>
   </property>
   <property name="toolTip">
    <string>Load CognitiveMap</string>
   </property>
  </action>
  <action name="actionRun_visualisation">
   <property name="text">
    <string>Run visualisation</string>
   </property>
  </action>
  <action name="actionShow_Point_of_Origin">
   <property name="text">
    <string>Show Point of Origin</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/Undo-icon.png</normaloff>:/Undo-icon.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo PaintEvent</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/Redo-Icon.png</normaloff>:/Redo-Icon.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo PaintEvent</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionHLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/HLine.png</normaloff>:/HLine.png</iconset>
   </property>
   <property name="text">
    <string>HLine</string>
   </property>
   <property name="toolTip">
    <string>HLine</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSpeichern_cogmap">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/filesave.png</normaloff>:/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save CognitiveMap</string>
   </property>
   <property name="toolTip">
    <string>Save CognitiveMap</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_ffnen_cogmap">
   <property name="icon">
    <iconset resource="Ressource.qrc">
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>Load Cognitive Map</string>
   </property>
   <property name="toolTip">
    <string>Load Cognitive Map</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Ressource.qrc"/>
 </resources>
 <connections/>
</ui>
